python/plugins/processing/gui/BatchPanel.py

```diff
@@ -178,9 +179,15 @@ def getWidgetFromParameter(self, param, row, col):
 
         return item
 
+    def getLastFolder():
+        settings = QSettings()
```

Comment:
QgsSettings

Reply:
yikes, that's from a different commit that adds the ability to remember the last folder for batch processes. Looks like i mixed things up. Let me re push this PR :-)

Reply:
Should be fine now